pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -189,7 +189,7 @@
     "Microsoft.Private.PackageBaseline": {
       "InboxOn": {}
     },
-    "Microsoft.VisualBasic": {
+    "Microsoft.VisualBasic.Core": {
```

Comment:
Undo and create new entry

Reply:
Should I keep the same contents for the new version?

---
In reply to: [241596047](https://github.com/dotnet/corefx/pull/34087#discussion_r241596047) [](ancestors = 241596047)

Reply:
New entry is only needed if we produce a package for the new assembly.  If we do it should get new contents, since it doesn't have any package history nor is it inbox anywhere.